src/shaders/symbol_text_and_icon.vertex.glsl

```diff
@@ -63,8 +63,8 @@ void main() {
         size = u_size;
     }
 
-    float h = elevation(a_pos);
-    vec4 projectedPoint = u_matrix * vec4(a_pos, h, 1);
+    vec3 h = elevationVector(a_pos) * elevation(a_pos);
```

Comment:
Modifications done to this shader file might not be complete. Have you found any render tests or use cases for this shader?

Reply:
@mpulkki-mapbox the test cases under `render-tests/text-field/` are covering this shader. As a proof of concept I set the projectedPoint value for zero and I got 15 failing tests there. So I would assume they're working correctly with the changes.